### DISCUSSION TITLE
Ability to close all tabs but the active one?

### DISCUSSION THREAD

**Discussion by taseppa** [mbnuqw/sidebery#1585] [General]
Moved recently from TreeStyle tabs plugin to this one and I wonder is there support for closing all tabs but the currently active one?

**Comment by mbnuqw** [comment]
Check the context menu editor (Sidebery settings > Menu editor > "Close other tabs") and Keybindings: "Close other tabs (in current panel)"

**Reply by taseppa** [reply]
Perfect, thanks
